tests/integ/test_session.py

```diff
@@ -570,7 +569,6 @@ def test_use_secondary_roles(session):
     session.use_secondary_roles(current_role[1:-1])
 
 
-@pytest.mark.skipif(IS_IN_STORED_PROC, reason="SP doesn't allow to close a session.")
 def test_close_session_twice(db_parameters):
```

Comment:
I ran the test in stored-proc environment and we still need to skip it. I didn't notice we were creating a new session - which is not allowed in SP. You were correct in updating the skip message earlier.